tm2/pkg/sdk/auth/keeper.go

```diff
@@ -156,6 +156,18 @@ func (ak AccountKeeper) GetNextAccountNumber(ctx sdk.Context) uint64 {
 	return accNumber
 }
 
+func (ak AccountKeeper) FeeCollectorAddress(ctx sdk.Context) crypto.Address {
+	feeCollector := ak.GetParams(ctx).FeeCollector
+	if feeCollector.IsZero() {
+		feeCollector = crypto.AddressFromPreimage([]byte(DefaultFeeCollectorName))
```

Comment:
By default it should be `std.DerivePkgAddr(gno.land/r/sys/txfees)` in the auth params. If the param value is set to ex `""`, then it should default to the param default

Reply:
no, it's a tm2 module, the default shouldn't be about gno.

Reply:
You're right 🤦‍♂️